TempleApp/src/main/res/layout/insert_data.xml

```diff
@@ -103,7 +103,7 @@
                 android:layout_height="wrap_content"
                 android:layout_gravity="center"
                 android:layout_marginTop="45dp"
-                android:text="Insert" />
+                android:text="@string/insert" />
```

Comment:
Here it is `INSERT` not `Inserting your values`    please make sure that it does not disturb any behavior of app and please check whether all the functionalities are working or not

Reply:
I have corrected that